OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -25,6 +25,7 @@ public class ProjectileArgs
 		public int[] InaccuracyModifiers;
 		public int[] RangeModifiers;
 		public int Facing;
+		public Func<int> CurrentFacing;
```

Comment:
Why do you need to duplicate? Unless I'm overlooking something, we don't have a case where the initial facing should be still used after firing, so the Facing alone should be enough for being used for this.

Reply:
That's a good point, and any callers can still use Facing() if a static facing is needed.